packages/story-editor/src/app/api/apiProvider.tsx

```diff
@@ -17,19 +17,22 @@
 /**
  * External dependencies
  */
-import PropTypes from 'prop-types';
 import { useCallback, useRef } from '@googleforcreators/react';
-import { getAllTemplates } from '@googleforcreators/templates';
+import type { ReactNode } from 'react';
+import { getAllTemplates, Template } from '@googleforcreators/templates';
 
 /**
  * Internal dependencies
  */
 import { useConfig } from '../config';
 import Context from './context';
 
-function APIProvider({ children }) {
+interface APIProviderProps {
+  children: ReactNode;
+}
+function APIProvider({ children }: APIProviderProps) {
```

Comment:
Let's use `PropsWithChildren<{}>` instead. NB: In React 18 it has a default empty type argument, but not in 17 unfortunately.

Reply:
We'll have to use `PropsWithChildren<Record<string, never>>` then since `{}` is not allowed.